rega-genotype/src/main/java/rega/genotype/utils/Settings.java

```diff
@@ -187,10 +187,13 @@ public static Settings getInstance(ServletContext context) {
         if (configFile == null) {
             String osName = System.getProperty("os.name");
             osName = osName.toLowerCase();
+            configFile = "etc/development-config.xml";
```

Comment:
I put the configuration file within the project folder is wiser to do it this way so there must always be looking for which way point, because it is an XML file.